api/views.py

```diff
@@ -160,3 +164,25 @@ def test(request):
         "status": "success",
         "payload": "Setup successful!"
     }, status=200)
+
+
+@api_view(["GET"])
+def rpms_images_fetcher_view(request):
+    release = request.query_params.get("release", None)
+
+    if release is None:
+        return Response(data={"status": "error", "message": "Missing \"release\" params in the url."})
+
+    # Always fetch data
+    try:
+        result = fetch_data(release)
```

Comment:
it should be `rpms_images_fetcher.fetch_data`

Reply:
You are right, I am sorry..